lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1101,16 +1129,28 @@ class LokiPushApiEndpointJoined(EventBase):
     """Event emitted when Loki joined."""
 
 
+class LokiPushApiAlertRulesError(EventBase):
+    """Event emitted if there is an error with Alert Rules."""
+
+
+class LokiPushApiAlertRulesOK(EventBase):
+    """Event emitted if there is no error with Alert Rules."""
+
+
 class LokiPushApiEvents(ObjectEvents):
     """Event descriptor for events raised by `LokiPushApiProvider`."""
 
     loki_push_api_endpoint_departed = EventSource(LokiPushApiEndpointDeparted)
     loki_push_api_endpoint_joined = EventSource(LokiPushApiEndpointJoined)
+    loki_push_api_alert_rules_error = EventSource(LokiPushApiAlertRulesError)
+    loki_push_api_alert_rules_ok = EventSource(LokiPushApiAlertRulesOK)
```

Comment:
I have no strong opinions about this.
Having only one event is handy since charm authors will only observe this event, but in the other hand they will have to add logic in the event handler to decide what to do.

Whatever approach we choose, it would be good to keep it in all charms. 
What do you think @simskij ?

Reply:
I'm obviously biased, but my perspective is that a single event which contains `errors` or some other indication which can be logged/put into `juju status`, and an attribute for "everything is ok now/in the first place" is a nicer API than listening to `loki_push_api_alert_rules_ok`, which will (hopefully) always be true

Reply:
I think we should discuss this in a Café